examples/flutter_gallery/lib/gallery/item.dart

```diff
@@ -81,6 +81,13 @@ List<GalleryItem> _buildGalleryItems() {
       buildRoute: (BuildContext context) => const VideoDemo(),
     ),
     // Material Components
+    new GalleryItem(
+      title: 'Backdrop',
```

Comment:
Should this be called 'Navigation' or some such? I thought at first it was a demo about blur filters or something.

Reply:
Backdrop is what the people who came up with it call it.